pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -489,15 +489,23 @@ func (k *Kubernetes) ConfigEnvs(name string, service kobject.ServiceConfig) []ap
 // CreateKubernetesObjects generates a Kubernetes artifact for each input type service
 func (k *Kubernetes) CreateKubernetesObjects(name string, service kobject.ServiceConfig, opt kobject.ConvertOptions) []runtime.Object {
 	var objects []runtime.Object
+	var replica int
+	if opt.Replicas > 1 {
```

Comment:
Yes, it is a boolean flag. We are checking if the default value has been changed.